src/script/E2EIdentity/E2EIdentityEnrollment.ts

```diff
@@ -88,10 +88,11 @@ export class E2EIHandler extends TypedEventEmitter<Events> {
 
   private createOIDCService() {
     const key = this.core.key;
-    if (!key) {
+    const targetURL = OIDCServiceStore.get.targetURL();
+    if (!key || !targetURL) {
       throw new Error('encryption key not set');
```

Comment:
Error Message is not targeted anymore.

Reply:
fixed with https://github.com/wireapp/wire-webapp/pull/16782/commits/c1f670a8fdc876a76a29444770aaa42b9456c4b9